### PR TITLE
Adding TagSpaces

### PR DESCRIPTION
**Pull request by uggrock**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:
-  [x] Your submissions are formatted according to the guidelines. 
  
  `[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language``
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

**Comment by nodiscc** [comment]
> You have searched the repository for any relevant issues or PRs.

https://github.com/Kickball/awesome-selfhosted/issues?utf8=%E2%9C%93&q=tagspaces%20 -> #235

**Comment by uggrock** [comment]
TagSpaces has a webdav edition, which can be self-hosted, please see the following link for more details 
https://www.tagspaces.org/blog/webdav-edition/
The linked [demo](demo.tagspaces.org), shows this self-hosted version in action. (Username: demo, password: demo )

**Comment by nodiscc** [comment]
Interesting, TagSpaces can use a WebDAV share as file storage (can be seen in action at http://demo.tagspaces.org/). I will reopen this PR now and let other maintainers decide. A few remarks:

 * The description should emphasize WebDAV compatibility in some way.
 * Note Taking and Editors is probably not the best section for this entry.

Thanks for bringing this up.

**Comment by AndyR207** [comment]
@uggrock Please update this to reflect @nodiscc's suggestions or this will be closed in 3 days.

File: README.md [hunk 1]
```diff
@@ -1099,6 +1099,7 @@ See also [Documentation Generators](#documentation-generators), [Wikimatrix](htt
   * [Sandstorm](https://sandstorm.io/) - Personal server for running self-hosted apps easily and securely. ([Demo](https://demo.sandstorm.io/), [Source Code](https://github.com/sandstorm-io/sandstorm)) `Apache` `C++/Other`
   * [sovereign](https://github.com/sovereign/sovereign) - A set of Ansible playbooks to build and maintain your own private cloud: email, calendar, contacts, file sync, IRC bouncer, VPN, and more. `GPLv3` `YAML/Other`
   * [Syncloud](http://syncloud.org/) - Your own online file storage, social network or email server. ([Source Code](https://github.com/syncloud/platform)) `GPLv3/Other` `Python/Other`
+  * [TagSpaces](https://www.tagspaces.org/) - TagSpaces is an offline, cross-platform file manager and organiser that also can function as a note taking app, providing an open-source alternative to Evernote. The WebDAV version of the application can be installed on top of a WebDAV servers such as Nextcloud or ownCloud. ([Demo](http://demo.tagspaces.org)), ([Source Code](https://github.com/tagspaces/tagspaces)) `AGPLv3` `Javascript`
```
Comment (AndyR207) [inline README.md]: Description is too long, 250 characters maximum please.

Demo & Source Code links are formatted wrong. Correct format should be `([Demo](url), [Source Code](url))`

**Comment by AndyR207** [comment]
@nodiscc please can you approve your change request and then assign n8225 and Kickball for merging.

**Comment by nodiscc** [comment]
Done.

**Pull request merged**